examples/device-allocator.cpp

```diff
@@ -56,17 +56,15 @@ int main(int argc, char const* argv[])
   auto device_allocator = umpire::make_device_allocator(allocator, sizeof(double), "my_device_alloc");
   // _sphinx_tag_make_dev_allocator_end
 
-  // Checking that a DeviceAllocator exists...
-  if (umpire::is_device_allocator(0)) {
+  // Checking that the DeviceAllocator just created can be found...
+  if (umpire::is_device_allocator(1)) {
```

Comment:
Please replace this with a call to get the allocator by name instead of an integer literal.

Also, the integer literal above needs to be negative.